doctor/tests.py

```
@@ -0,0 +1,188 @@
+            medical_system_code='DOC12345',
+            specialty='general',
+            phone_number='09123456789',
+            is_verified=True
+        )
+        
+        url = reverse('doctor:search_doctors')
+        response = self.client.get(url, {'specialty': 'general'})
+        
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+    
+    def test_unauthorized_access(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_unauthorized_access`

Add return type annotation: `None`

(ANN201)